bambi/models.py

```diff
@@ -763,15 +807,37 @@ def prior_predictive(self, draws=500, var_names=None, omit_offsets=True, random_
 
         return idata
 
+    @overload
     def predict(
         self,
-        idata,
-        kind="mean",
-        data=None,
-        inplace=True,
-        include_group_specific=True,
-        sample_new_groups=False,
-    ):
+        idata: InferenceData,
+        kind: Literal["mean", "pps"] = "mean",
+        data: Optional[pd.DataFrame] = None,
+        inplace: Literal[True] = True,
```

Comment:
Shouldn't this be `bool` instead of `Literal[True]`? I'm not a typing expert so I may be missing some details, but since this can be either `True` or `False`, I think it should be `bool`